frontend/src/App.tsx

```diff
@@ -1,13 +1,17 @@
 import React from 'react';
 import { BrowserRouter, Route, Routes } from 'react-router-dom';
-import HomeScreen from 'screens/HomeScreen';
+import Map from 'components/Map/index';
+import NavBar from 'Header/NavBar';
+import Footer from 'Footer/Footer';
 
 function App() {
   return (
     <BrowserRouter>
+      <NavBar />
       <Routes>
-        <Route path="/" element={<HomeScreen />} />
+        <Route path="/Map" element={<Map />} />
```

Comment:
I think tat the path should be "/" insted of "Map". But we need to discuss it with team.